tests/genai/scorers/deepeval/conftest.py

```diff
@@ -0,0 +1,4 @@
+import pytest
+
+# Skip all tests in this directory if deepeval is not installed
+pytest.importorskip("deepeval", reason="deepeval is not installed")
```

Comment:
Can we remove this file and make sure tests are tested by installing deepeval in tests?

Reply:
We need to make sure this folder is tested right? Otherwise there's no point on writing tests, it should be run in [genai job](https://github.com/mlflow/mlflow/actions/runs/19663137613/job/56414134578). Can you try installing inside this job?